docs/topics/install/upgrading.md

```diff
@@ -1,27 +1,49 @@
 # Upgrading Ambassador Edge Stack
 
-Since Ambassador Edge Stack's configuration is entirely stored in Kubernetes resources, no special process is necessary to upgrade Ambassador Edge Stack. If you're using the YAML files supplied by Datawire, you'll be able to upgrade simply by repeating the following `kubectl apply` commands.
-
-First, determine if Kubernetes has RBAC enabled:
+Since Ambassador Edge Stack's configuration is entirely stored in Kubernetes resources, no special process
+is necessary to upgrade Ambassador Edge Stack.
+
+The steps to upgrade depend on the method that used used to install Ambassador Edge Stack, as indicated below.
+
+* If you installed using the Operator, then you'll need to [use the Operator to perform the upgrade](https://www.getambassador.io/docs/latest/topics/install/aes-operator/#updates-by-the-operator).
```

Comment:
Please make sure referenced to other documentation pages are relative, i.e., 'aes-operator/#updates-by-the-operator' in this case.